// node-red-contrib-twitter-follow/env.js
module.exports = {
    consumer_key: 'LiOhSvQsDNghdMzLY8yiWm6ns', // consumer keyを記入
    consumer_secret     : 'SDgfxNDPPIgOilbx8Ylx7ieR5i3LmcWjc530lVhi086Us3JqEi', // consumer secretを記入
    access_token_key    : '1093137096791867392-4YyE9sbeqkdk2zyvIvMo2hxHlRGFr6', // access tokenを記入
    access_token_secret : '2yFOfnc4Xz4FNcgvsZpc1iZ0lZ5bIWGT4oEsjaFQwVwO1' // access token secretを記入
}

/**
 
```html
<script type="text/javascript">
    RED.nodes.registerType('LINE Reply Message',{
        category: 'LINE',
        color: '#01B301',
        defaults: {
            name: {value:""},
            channelAccessToken:{value:""},
            channelSecret:{value:""},
            replyMessage:{value:"Hello!"}
        },
        inputs:1,
        outputs:1,
        icon: "arrow-in.png",
        align: "right",
        label: function() {
            return this.name||"LINE Reply Message";
        }
    });
</script>

<script type="text/x-red" data-template-name="LINE Reply Message">
    <div class="form-row">
        <label for="node-input-name"><i class="icon-tag"></i> Name</label>
        <input type="text" id="node-input-name" placeholder="Name">
    </div>
    <div class="form-row">
        <label for="node-input-channelAccessToken"><i class="icon-tag"></i> AccessToken</label>
        <input type="text" id="node-input-channelAccessToken" placeholder="ChannelAccessToken">
    </div>
    <div class="form-row">
        <label for="node-input-channelSecret"><i class="icon-tag"></i> Secret</label>
        <input type="password" id="node-input-channelSecret" placeholder="ChannelSecret">
    </div>
    <div class="form-row">
        <label for="node-input-replyMessage"><i class="icon-tag"></i> ReplyMessage</label>
        <input type="text" id="node-input-replyMessage" placeholder="ReplyMessage">
    </div>
</script>

<script type="text/x-red" data-help-name="LINE Reply Message">
    <p>ノードで指定した文字列をLINEに返します</p>
</script>
```

 */
